tests/fiuinfo_test.cc

```diff
@@ -0,0 +1,32 @@
+#include <gtest/gtest.h>
+#include <unistd.h>
+
+#include "utilities/fault_injection.h"
+#include "utilities/utils.h"
+
+std::string fiu_script;
+
+TEST(Fiuinfo, PassInfoWithRun) {
+  EXPECT_NE(fiu_fail("fail"), 0);
+  EXPECT_EQ(fault_injection_last_info(), "test");
+}
+
+TEST(Fiuinfo, PassInfoWithCtrl) {
+  EXPECT_EQ(fiu_fail("failctrl"), 0);
+  Utils::shell(fiu_script + " ctrl -c 'enable name=failctrl,failinfo=test_ctrl' " + std::to_string(getpid()), nullptr);
+  EXPECT_NE(fiu_fail("failctrl"), 0);
```

Comment:
As discussed, I think checking that we can unset the failure and recover is worthwhile.

Reply:
Should be covered now.